### PR TITLE
Template for Issue and Source Request

### PR DESCRIPTION
**Pull request by OshekharO**
Informative template for issue and source request.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/request.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: ⭐ Extension request
+description: Suggest a extension that you would like see added
+labels: []
```
Comment (rhenwinch) [inline .github/ISSUE_TEMPLATE/request.yml:3]: Could you add the following label: provider request

**Review by rhenwinch — CHANGES_REQUESTED** [review]
Hi!

Thanks for the initiative wouldn't even think about creating github configs for this repo.

Anyway, could you re-phrase the 'extension' and 'source' words to 'provider' instead?

**Comment by OshekharO** [comment]
> Hi!
> 
> Thanks for the initiative wouldn't even think about creating github configs for this repo.
> 
> Anyway, could you re-phrase the 'extension' and 'source' words to 'provider' instead?

It just make the whole thing user-friendly and also easy for developers to get whole info

**Review by OshekharO — COMMENTED** [review]
Done

File: .github/ISSUE_TEMPLATE/issue.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
+name: 🐞 Issue report
+description: Report a provider issue with a source
+labels: [ bug ]
+body:
+
+    -   type: input
+        id: source
+        attributes:
+            label: Extension information
```
Comment (rhenwinch) [inline .github/ISSUE_TEMPLATE/issue.yml:9]: Provider information

**Review by rhenwinch — APPROVED** [review]

**Pull request merged**